### DISCUSSION TITLE
haproxy.service not active solution

### DISCUSSION THREAD

**Discussion by yanqingfu** [Seagate/cortx-s3server#357] [Show and tell]
[root@ssc-vm-1061 cortx-s3server]# ./jenkins-build.sh --skip_build --skip_tests
Skip build step
Skip test step
++ pwd
+ S3_BUILD_DIR=/var/cortx/cortx-s3server
+ ANSIBLE_DIR=/var/cortx/cortx-s3server/scripts/env/dev
+ USE_SUDO=
+ [[ 0 -ne 0 ]]
+ [[ 0 -eq 1 ]]
+ source /root/.s3_ldap_cred_cache.conf
++ ldap_root_pwd=seagateseagatseagate
++ ldap_admin_pwd=ldapadmin
+ '[' '' '!=' '' ']'
+ export PATH=/opt/seagate/cortx/s3/bin/:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt    /puppetlabs/bin:/root/bin
+ PATH=/opt/seagate/cortx/s3/bin/:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppet    labs/bin:/root/bin
+ echo /opt/seagate/cortx/s3/bin/:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppet    labs/bin:/root/bin
/opt/seagate/cortx/s3/bin/:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bi    n:/root/bin
+ ulimit -c unlimited
+ rpm -q haproxy
haproxy-1.5.18-9.el7.x86_64
+ systemctl status haproxy
**● haproxy.service - HAProxy Load Balancer
   Loaded: loaded (/usr/lib/systemd/system/haproxy.service; disabled; vendor preset: disabled)
   Active: inactive (dead)**
[root@ssc-vm-1061 cortx-s3server]# modprobe lnet
[root@ssc-vm-1061 cortx-s3server]# lctl list_nids
192.168.47.113@tcp
10.230.247.0@tcp1

solution: 
[root@ssc-vm-1061 cortx-s3server]# `**systemctl restart haproxy**`

[root@ssc-vm-1061 cortx-s3server]# ./jenkins-build.sh --skip_build --skip_tests
Skip build step
Skip test step
++ pwd
+ S3_BUILD_DIR=/var/cortx/cortx-s3server
+ ANSIBLE_DIR=/var/cortx/cortx-s3server/scripts/env/dev
+ USE_SUDO=
+ [[ 0 -ne 0 ]]
+ [[ 0 -eq 1 ]]
+ source /root/.s3_ldap_cred_cache.conf
++ ldap_root_pwd=seagateseagatseagate
++ ldap_admin_pwd=ldapadmin
+ '[' '' '!=' '' ']'
+ export PATH=/opt/seagate/cortx/s3/bin/:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt    /puppetlabs/bin:/root/bin
+ PATH=/opt/seagate/cortx/s3/bin/:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppet    labs/bin:/root/bin
+ echo /opt/seagate/cortx/s3/bin/:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppet    labs/bin:/root/bin
/opt/seagate/cortx/s3/bin/:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bi    n:/root/bin
+ ulimit -c unlimited
+ rpm -q haproxy
haproxy-1.5.18-9.el7.x86_64
+ systemctl status haproxy
● haproxy.service - HAProxy Load Balancer
   Loaded: loaded (/usr/lib/systemd/system/haproxy.service; disabled; vendor preset: disabled)
   Active: active (running) since Thu 2020-10-29 05:59:58 MDT; 6s ago
 Main PID: 5293 (haproxy-systemd)
   CGroup: /system.slice/haproxy.service
           ├─5293 /usr/sbin/haproxy-systemd-wrapper -f /etc/haproxy/haproxy.cfg -p /run/haproxy.pid
           ├─5294 /usr/sbin/haproxy -f /etc/haproxy/haproxy.cfg -p /run/haproxy.pid -Ds
           ├─5295 /usr/sbin/haproxy -f /etc/haproxy/haproxy.cfg -p /run/haproxy.pid -Ds
           └─5296 /usr/sbin/haproxy -f /etc/haproxy/haproxy.cfg -p /run/haproxy.pid -Ds

**Comment by johnbent** [comment]
Hey @nileshgovande , can you help here please?

**Comment by nileshgovande** [comment]
haproxy running is a requirement as of now. @Saumya-Sunder - Pls document accordingly as of now.

We will try and decouple.

Tagging @bkirunge7

**Reply by Saumya-Sunder** [reply]
So @nileshgovande this goes into prereqs?

**Comment by Saumya-Sunder** [comment]
Addressed the issue here https://github.com/Seagate/cortx-s3server/pull/458
